firmware/src/mcu/samd11/usb_descriptors.c

```diff
@@ -40,9 +40,9 @@ tusb_desc_device_t const desc_device =
 
 	// Use Interface Association Descriptor (IAD) for CDC
 	// As required by USB Specs IAD's subclass must be common class (2) and protocol must be IAD (1)
```

Comment:
Could you please update this comment? I would like it to say that we are using `bDeviceClass = 0` to indicate that we are a composite device and not using Interface Association Descriptors.